src/overlays/actors/ovl_Item_B_Heart/z_item_b_heart.h

```diff
@@ -3,12 +3,20 @@
 
 #include "global.h"
 
-struct ItemBHeart;
+typedef enum {
+    /*   0 */ BHEART_PARAM_NORMAL,     // Spawn Parameter for Normal Scale.
+    /*  35 */ BHEART_PARAM_SMALL = 35, // Spawn Parameter for Small Scale.
+} ItemBHeartParams;
+
+#define BHEART_SCALE_SMALL (0.1f)           // Scale for Small Mode
+#define BHEART_SCALE_NORMAL (1.0f)          // Scale value for Normal Mode
+#define BHEART_SCALE_MIN_COLLECTIBLE (0.5f) // unitScale <50% will disable collection
 
 typedef struct ItemBHeart {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ UNK_TYPE1 unk_144[0x24];
-    /* 0x168 */ f32 unk_168;
+    /* 0x144 */ UNK_TYPE1 unk_144[0x20];
+    /* 0x164 */ f32 variableScale;
+    /* 0x168 */ f32 scale;
```

Comment:
What do you think about calling this `unitScale`? You used that in the comment above, and I like how it differentiates it from the *actor's* `scale` variable.

Reply:
`unitScale` makes me think it always has to be `1.0f` (i.e. a unit value). What about `baseScale`?

Reply:
Just want to bump on engineer's comment

Reply:
baseScale is fine, will rename. (TY again for the poke hensldm)